specification/DigitalOcean-public.v2.yaml

```
@@ -581,6 +581,10 @@ paths:
     put:
       $ref: 'resources/apps/apps_update.yml'
 
+  /v2/apps/{app_id}/components/{component_name}/logs:
+    get:
+      $ref: 'resources/apps/apps_get_logsActiveDeployment.yml'
```

this is very minor- but could you convert this to be all snakecase? 
```suggestion
      $ref: 'resources/apps/apps_get_logs_active_deployment.yml'
```

I know we have other resources that follow the snake_case_camelCase pattern, but we're trying to move away from that.